lib/system/zephyr/mutex.h

```diff
@@ -49,6 +49,7 @@ extern "C" {
 
 typedef struct k_sem metal_mutex_t;
 
+#define METAL_MUTEX_INIT(m) _K_SEM_INITIALIZER(m, 1, 1)
 #define METAL_MUTEX_DEFINE(m) K_SEM_DEFINE(m, 1, 1)
```

Comment:
Just want to know what's the difference METAL_MUTEX_INIT and METAL_MUTEX_DEFINE?

Reply:
METAL_MUTEX_INIT is for when we have a struct element we need to initialize.  META_MUTEX_DEFINE is for a singleton global or static.  I've updated the comments and commit messages that will hopefully help.